examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -125,51 +128,118 @@ bool OTAProviderExample::SelectOTACandidate(const uint16_t requestorVendorID, co
     return candidateFound;
 }
 
+UserConsentSubject OTAProviderExample::GetUserConsentSubject(const app::CommandHandler * commandObj,
+                                                             const app::ConcreteCommandPath & commandPath,
+                                                             const QueryImage::DecodableType & commandData, uint32_t targetVersion)
+{
+    UserConsentSubject subject;
+    FabricIndex fabricIndex = commandObj->GetSubjectDescriptor().fabricIndex;
+    subject.fabricId        = Server::GetInstance().GetFabricTable().FindFabricWithIndex(fabricIndex)->GetFabricId();
```

Comment:
Fabric ID without root public key is ambiguous. Suggest storing the root public key as well, or remove fabricId and store only fabricIndex, but then there is a dependency on the fabric table elsewhere (which is likely fine)

Reply:
Using fabric index